messaginginapp/src/main/java/io/customer/messaginginapp/gist/utilities/ModalAnimationUtil.kt

```diff
@@ -0,0 +1,66 @@
+package io.customer.messaginginapp.gist.utilities
+
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.graphics.Color
+import android.view.View
+import androidx.annotation.ColorInt
+import io.customer.sdk.core.di.SDKComponent
+
+internal object ModalAnimationUtil {
+
+    const val FALLBACK_COLOR_STRING = "#33000000"
+
+    private const val TRANSLATION_ANIMATION_DURATION = 150L
+    private const val ALPHA_ANIMATION_DURATION = 150L
+    private const val COLOR_ANIMATION_DURATION = 100L
+
+    private val logger = SDKComponent.logger
+
+    fun createAnimationSetInFromTop(target: View, overlayEndColor: String): AnimatorSet {
+        return createEnterAnimation(target, overlayEndColor, -100f)
```

Comment:
```suggestion
        target.alpha = 0f
        target.translationY = -100f
        return createEnterAnimation(target, overlayEndColor, -100f)
```

We can prevent frame drop here by setting initial state